src/components/OrderedListItem/index.tsx

```diff
@@ -0,0 +1,44 @@
+import {
+	Avatar,
+	Box,
+	ListItem,
+	ListItemIcon,
+	ListItemProps,
+	Typography,
+} from '@mui/material';
+
+export interface OrderedListItemProps extends ListItemProps {
+	index: number;
+}
+
+export const OrderedListItem: React.FC<OrderedListItemProps> = ({
+	index,
+	children,
+	...orderedListItemProps
+}) => {
+	return (
+		<ListItem
+			sx={{ pl: 0, ...orderedListItemProps.sx }}
+			{...orderedListItemProps}
```

Comment:
Doesn't this make it so that the sx is overriden by the subsequent `{...orderedListItemProps}` if it has its own `sx` since we did not destructure the `sx` from it?
```suggestion
			sx={{ pl: 0, ...orderedListItemProps.sx }}
			{...omit(orderedListItemProps, 'sx')}
```
(preferably just destructure the sx above, I just used the `omit` to communicate what I am taking about :P)